dev/dialog.html

```diff
@@ -5,26 +5,34 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Dialog</title>
-    <script type="module" src="./common.js"></script>
 
     <script type="module">
+      import './common.js'; // A separate <script type="module"> for this import crashes Safari often
       import '@vaadin/dialog';
+      import '@vaadin/button';
+      import '@vaadin/icon';
+      import '@vaadin/vaadin-lumo-styles/iconset';
     </script>
   </head>
 
   <body>
     <vaadin-dialog></vaadin-dialog>
 
-    <button>Open dialog 1</button>
+    <vaadin-button>Open dialog 1</vaadin-button>
 
     <script type="module">
       const dialog1 = document.querySelector('vaadin-dialog');
+      dialog1.headerTitle = 'Title';
+      dialog1.draggable = true;
+      dialog1.resizable = true;
 
       dialog1.renderer = (root) => {
```

Comment:
I wonder if it would make sense to use Lit at least for the renderers for readability purposes? It is quite hard to read the code for such a sophisticated dialog.

Reply:
I'm going to add Lit helpers to my CF project later: https://github.com/web-padawan/lit-vaadin-helpers

We'll eventually move them to the monorepo at some point, as part of #266

Reply:
Okay 👍 but my point was mainly about the current dialog dev page where Lit could be used to make the renderers more declarative.